gdal/data/FGD_GMLSchema.xsd

```diff
@@ -0,0 +1,842 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xs:schema targetNamespace="http://fgd.gsi.go.jp/spec/2008/FGD_GMLSchema"
```

Comment:
Uhmm, exactly we should check license. It is not mandatory to include all in gdal/data. I'd like to remove it in next commit.